cypress/integration/app/sideBarNavigation.js

```diff
@@ -0,0 +1,22 @@
+const SIDEBAR_NAVIGATION_URL = '/iframe.html?id=app-stories-sidebarnavigation--side-bar-navigation';
+const NAVIGATION_ITEM = '[data-id="sidebar-item-clickable-element"]';
+
+describe('SideBarNavigation', () => {
+    beforeEach(() => {
+        cy.visit(SIDEBAR_NAVIGATION_URL);
+    });
+
+    it('should navigate to the selected route', () => {
+        cy.get(NAVIGATION_ITEM).eq(1).click();
+        cy.location().should((loc) => {
+            expect(loc.pathname).to.equal('/export');
+        });
+    });
+
+    it('should select an option based on the browser url', () => {
+        cy.window().then((win) => {
```

Comment:
can we use async/await syntax here?

Reply:
This is not possible as cypress commands does not return Promises.
https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Commands-Are-Promises